2024/half-2/day-15/function-length/function-length.ts

```
@@ -0,0 +1,3 @@
+export default function functionLength(fn: Function): number {
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)